client/src/pages/welcome-grasshopper.js

```diff
@@ -0,0 +1,259 @@
+            size='9x'
+          />
+          <h2 className='black-text'>HTML5</h2>
+        </Col>
+        <Col md={3} sm={6} xs={12}>
+          <FontAwesomeIcon
+            className='landing-skill-icon'
+            icon={faCss3Alt}
+            size='9x'
+          />
+          <h2 className='black-text'>CSS3</h2>
+        </Col>
+        <Col md={3} sm={6} xs={12}>
+          <FontAwesomeIcon
+            className='landing-skill-icon'
+            icon={faJs}
+            size='9x'
+          />
+          <h2 className='black-text'>JavaScript</h2>
+        </Col>
+        <Col md={3} sm={6} xs={12}>
+          <FontAwesomeIcon
+            className='landing-skill-icon'
+            icon={faDatabase}
+            size='9x'
+          />
+          <h2 className='black-text'>Databases</h2>
+        </Col>
+      </Row>
+      <Row className='text-center'>
+        <Col md={3} sm={6} xs={12}>
+          <FontAwesomeIcon
+            className='landing-skill-icon'
+            icon={faGithub}
+            size='9x'
+          />
+          <h2 className='black-text'>Git & GitHub</h2>
+        </Col>
+        <Col md={3} sm={6} xs={12}>
+          <FontAwesomeIcon
+            className='landing-skill-icon'
+            icon={faNodeJs}
+            size='9x'
+          />
+          <h2 className='black-text'>Node.js</h2>
+        </Col>
+        <Col md={3} sm={6} xs={12}>
+          <FontAwesomeIcon
+            className='landing-skill-icon'
+            icon={faReact}
+            size='9x'
+          />
+          <h2 className='black-text'>React.js</h2>
+        </Col>
+        <Col md={3} sm={6} xs={12}>
+          <Image
+            alt='The D3.js Logo'
+            className='landing-skill-icon custom-landing-skill-icon'
+            src='https://s3.amazonaws.com/freecodecamp/d3-logo.svg'
+          />
+          <h2 className='black-text'>D3.js</h2>
+        </Col>
+      </Row>
+      <hr />
+      <Spacer />
+      <h3>
+        freeCodeCamp is a donor-supported tax-exempt 501(c)(3) nonprofit
+        organization (United States Federal Tax Identification Number:
+        82-0779546)
+      </h3>
+      <p className='large-p'>
+        Our mission: to help people learn to code for free. We accomplish this
+        by creating thousands of videos, articles, and interactive coding
+        lessons - all freely available to the public. We also have thousands of
+        freeCodeCamp study groups around the world.
+      </p>
+      <p className='large-p'>
+        Donations to freeCodeCamp go toward our education initiatives, and help
+        pay for servers, services, and staff. You can{' '}
+        <a
+          className='large-p underlined-link'
+          href='https://donate.freecodecamp.org/'
+          rel='noopener noreferrer'
+          target='_blank'
+          >
+          make a tax-deductible donation here
+        </a>
+      </p>
+      .<Spacer />
+      <BigCallToAction />
+      <Spacer />
```

Comment:
Refactor this and use the size property instead as suggested in another comment below.